Jobs.Fetcher.TikTok/Helpers/ApiDataFetcher.cs

```diff
@@ -0,0 +1,242 @@
+                    Type = tag["type"].ToObject<int>(),
+                    UserId = tag["userId"].ToString(),
+                    IsCommerce = tag["isCommerce"].ToObject<bool>(),
+                    UserUniqueId = tag["userUniqueId"].ToString(),
+                    SecureUId = tag["secUid"].ToString(),
+                    SubType = tag["subType"].ToObject<int>()
+                });
+            }
+            return newTags;
+        }
+
+        public static List<EffectSticker> GetTikTokEffectStickersFromJSON(JToken effectStickersJSON) {
+            var effectStickers = new List<EffectSticker>();
+            if (effectStickersJSON == null) {
+                return effectStickers;
+            }
+            foreach (var sticker in effectStickersJSON) {
+                var newEffectSticker = new EffectSticker() {
+                    Name = sticker["name"].ToString(),
+                    Id = sticker["ID"].ToString()
+                };
+                effectStickers.Add(newEffectSticker);
+            }
+            return effectStickers;
+        }
+
+        public static Author GetTikTokAuthorFromAuthorJSON(JToken authorJSON) {
+            return new Author() {
+                       Id = authorJSON["id"].ToString(),
+                       UniqueId = authorJSON["uniqueId"].ToString(),
+                       Nickname = authorJSON["nickname"].ToString(),
+                       AvatarThumbnail = authorJSON["avatarThumb"].ToString(),
+                       AvatarMedium = authorJSON["avatarMedium"].ToString(),
+                       AvatarLarge = authorJSON["avatarLarger"].ToString(),
+                       Signature = authorJSON["signature"].ToString(),
+                       Verified = authorJSON["verified"].ToObject<bool>(),
+                       SecurityUId = authorJSON["secUid"].ToString(),
+                       Secret = authorJSON["secret"].ToObject<bool>(),
+                       FTC = authorJSON["ftc"].ToObject<bool>(),
+                       Relation = authorJSON["relation"].ToObject<int>(),
+                       OpenFavorite = authorJSON["openFavorite"].ToObject<int>(),
+                       CommentSetting = authorJSON["commentSetting"].ToObject<int>(),
+                       DuetSetting = authorJSON["duetSetting"].ToObject<int>(),
+                       StitchSetting = authorJSON["stitchSetting"].ToObject<int>(),
+                       PrivateAccount = authorJSON["privateAccount"].ToObject<bool>()
+            };
+        }
+
+        public static Author GetTikTokAuthorFromPostJSON(JToken postJSON) {
+            return new Author() {
+                       Id = postJSON["authorId"].ToString(),
+                       UniqueId = postJSON["author"].ToString(),
+                       Nickname = postJSON["nickname"].ToString()/*,
+                                                                    AvatarThumbnail = authorJSON["avatarThumb"].ToString(),
```

Comment:
Will this get added later?

Reply:
I forgot to remove this. This is the all the author data available when the Json file is incomplete